boards/arm/arty/board.c

```diff
@@ -58,9 +58,9 @@ int board_daplink_qspi_mux_select(enum board_daplink_qspi_mux_mode mode)
 bool board_daplink_is_fitted(void)
 {
 	/*
-	 * The DAPLINK_fitted_n signal is routed an IRQ line. It is used as a
-	 * level-detect non-interrupt signal to determine if the DAPLink
-	 * shield is fitted.
+	 * The DAPLINK_fitted_n signal is routed to an IRQ line. It is used as a
+	 * level-detect non-interrupt signal to determine if the DAPLink shield
+	 * is fitted.
```

Comment:
This should be moved to first commit

Reply:
@erwango Thank you for reviewing. Good catch! I must have performed a fixup on the wrong commit at some point. Fixed.